src/nyc_trees/js/src/reserveBlockfacePage.js

```diff
@@ -3,7 +3,7 @@
 var $ = require('jquery'),
     L = require('leaflet'),
     mapModule = require('./map'),
-    zoom = require('./mapUtil').zoom,
+    zoom = require('./mapUtil').ZOOM,
```

Comment:
Since we added `mapUtil` it looks like this `zoom` dependency is no longer needed on most pages.